packages/plugin-gamepad/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@Shaobin-Jiang/jspsych-gamepad",
```

Comment:
To publish this in our npm namespace this should be `"@jspsych-contrib/plugin-gamepad"`